### PR TITLE
Additional features for IBM X-force plug-in

### PR DESCRIPTION
**Pull request by gekkeharry13**
Hi guys, as we were struggling during testing this plug-in, I added certificate verification options. And to actually use the plug-in we require proxy support. So hereby a pull request for adding both these features.

If there is anything not compliant to any standards, let me know so I can adjust it.

### CODE REVIEW

File: analyzers/IBMXForce/IBMXForce_Lookup.json [hunk 1]
```diff
@@ -37,6 +37,13 @@
       "required": true,
       "multi": false,
       "type": "string"
+    },
+    {
+      "name": "verify",
+      "description": "Enable/Disable certificate verification",
+      "required": false,
+      "multi": false,
+      "type": "string"
```
Comment (nadouani) [inline analyzers/IBMXForce/IBMXForce_Lookup.json]: This can be ob type `boolean`, instead of making it a `string` and convert it to `boolean`

Comment (nadouani) [reply analyzers/IBMXForce/IBMXForce_Lookup.json]: Could be 
```
{
      "name": "verify",
      "description": "Enable/Disable certificate verification",
      "type": "boolean",
      "required": false,
      "multi": false,
      "default": true
    }
```

Comment (nadouani) [reply analyzers/IBMXForce/IBMXForce_Lookup.json]: I'll fix this change

Comment (gekkeharry13) [reply analyzers/IBMXForce/IBMXForce_Lookup.json]: Hi Nabil, sorry I thought you already adjusted the suggestion. I did not know there was a boolean option (could not find it for some reason, probably my fault) If this is changed, then we can remove the convert to boolean part as well

```
    def str2bool(self, v):
         if isinstance(v, str) or isinstance(v, int) or isinstance(v, unicode) :
             return v.lower() in ("yes", "true", "t", "1")
         if isinstance(v, bool):
             return v
```
If you are busy, I can try to fix it today? 😄

Comment (nadouani) [reply analyzers/IBMXForce/IBMXForce_Lookup.json]: @gekkeharry13 oh don't worry it was not a big deal ;) thanks for the contribution

**Pull request merged**